static/js/services/delete-docs.js

```diff
@@ -60,13 +62,44 @@ var _ = require('underscore');
         });
       };
 
+      var bulkDeleteRemoteDocs = function (docs, eventListeners) {
```

Comment:
Could you document this please? Explaining at least the structure of docs and why it's a list of objects with _id in them and not just an array of strings (unless this is documented on the api side)

Reply:
I think the only reason is in case we want to add in a `_rev` parameter here to ensure people aren't accidentally deleting changes others have made, but I'm happy to just have it as an array for now and then change it up if and when we add in the requirement for `_rev`?

Reply:
Uhm well at least the api should be like that, so we don't have to incompatibly change it later. So there should be documentation on structure somewhere. I don't mind here if you document it or change it, as long as it's clear without reading old PRs :-)